src/dataflow/src/render/reduce.rs

```diff
@@ -7,6 +7,51 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+//! Reduction execution planning and dataflow construction.
+//!
+//! Materialize needs to be able to maintain reductions incrementally (roughly, using
+//! time proportional to the number of changes in the input) and ideally, with a
+//! memory footprint proportional to the number of reductions being computed. To achieve that,
+//! reduction functions are divied into three categories, each with their own specialized plan
```

Comment:
"divvied", weirdly enough.

Reply:
I'll be honest - the intent here was "divided" lol

Reply:
Even better!